values.yaml

```diff
@@ -41,6 +41,8 @@ global:
     # namespace pools: define k8s namespaces in which Astronomer will manage Airflow deployments
     namespacePools:
       enabled: false
+      # if rbacCreate is set to false then platform does not create roles and rolebinding.
+      rbacCreate: true
```

Comment:
This should probably be createRbac so it follows the same form as other variables we use, like `createJetStreamJob`, `createSecret`, `createDefaultIndex`, etc..

Reply:
done, changes to createRbac